### ISSUE TITLE
Can't login in lightroom v9.4.1

### ISSUE THREAD

**Issue by aleks01100001** [ReVanced/revanced-patches#3403]
### Bug description

After patching the Lightroom app with the 'Unlock premium' patch, the app won't sign in. It gets stuck on the loading screen after I add my account details.
![Screenshot_20240701_152019_Lightroom for Samsung](https://github.com/ReVanced/revanced-patches/assets/49501241/1dbbb8bc-7c5f-4c74-88de-93b07896eae1)

### Acknowledgements

- [X] I have checked all open and closed bug reports and this is not a duplicate.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Google login won't work for patched apps. For support, move to one of our links at https://revanced.app.

**Comment by aleks01100001** [comment]
@oSumAtrIX Yes, I know that Google login doesn't work. I tried to log in with my Adobe account.

**Comment by imzaki1994** [comment]
any updates?

**Comment by gabrielparca** [comment]
I don't have issues logging in on v9.4.3 — but then the screen shows the little loading wheel spinning for quite a while

![Screenshot_20240811_115853_Lightroom for Samsung.png](https://github.com/user-attachments/assets/4e13638d-bfa6-4913-a53d-d9af150a58b4)

until it displays the message:

```
`Couldn't complete purchase

Network error occurred. Try again
later.
IC100006/getProductsDetails/
com.adobe.lrmobile`

```
![Screenshot_20240811_115901_Lightroom for Samsung.png](https://github.com/user-attachments/assets/89d9503e-f547-4adf-a4fe-3dcbcf8f9812)

by pressing `Ok` this closes, showing a red one:

```
`⚠ Something went wrong.
Please try again after some time`

```
![Screenshot_20240811_114842_Lightroom for Samsung.png](https://github.com/user-attachments/assets/6a9b05a6-f6a8-40f7-9458-5513738832d2)

**Comment by Stenafo** [comment]
Some problems can be caused by the fact that from version 9.2.1 and later the downloadable apks are called: **Lightroom for Samsung**
These apks are probably not compatible with other phones.
Standard apks are only in the bundle version.

You can check in the two variants of this version and subsequent versions: https://www.apkmirror.com/apk/adobe/lightroom/lightroom-9-2-1-release/

**Comment by giwrgosmant** [comment]
> Some problems can be caused by the fact that from version 9.2.1 and later the downloadable apks are called: **Lightroom for Samsung**
> These apks are probably not compatible with other phones.
> Standard apks are only in the bundle version.
> 
> You can check in the two variants of this version and subsequent versions: https://www.apkmirror.com/apk/adobe/lightroom/lightroom-9-2-1-release/

I confirm.
Version 9.2 seems to work fine on my OP12 while other versions which says "for samsung" not.

**Comment by giwrgosmant** [comment]
> > > Some problems can be caused by the fact that from version 9.2.1 and later the downloadable apks are called: **Lightroom for Samsung**
> > > These apks are probably not compatible with other phones.
> > > Standard apks are only in the bundle version.
> > > You can check in the two variants of this version and subsequent versions: https://www.apkmirror.com/apk/adobe/lightroom/lightroom-9-2-1-release/
> > 
> > 
> > I confirm. Version 9.2 seems to work fine on my OP12 while other versions which says "for samsung" not.
> 
> Have you tried 9.5 [here](https://www.apkmirror.com/apk/adobe/lightroom/lightroom-9-5-0-release/lightroom-photo-video-editor-9-5-0-android-apk-download/)? This version is not for Samsung in the filename, though.

9.5 There is only "bundle" available in apkmirror, which cannot be patched with ReVanced Manager,

**Comment by aleks01100001** [comment]
> > I tried it too with antisplit-m, but it crashes. I don't think it's because of your phone, man.
> 
> Uhmm...maybe Adobe added some checks to make their app crashes if its tempered, dunno exactly.
> 
> 

I don't think so, because there are some modded pre-built APKs of Lightroom in v9.5.0.

**Comment by oSumAtrIX** [comment]
libpairip is protecting the app. You'd have to reverse engineer these modded apps and see what they did
